tracing-fmt/src/filter/mod.rs

```diff
@@ -28,3 +28,18 @@ where
         (self)(metadata, ctx)
     }
 }
+
+pub fn none() -> NoFilter {
```

Comment:
This was added so that `tracing-fmt`'s filtering can be disabled in the examples, but also I thought it was good to have.

Eventually we may just want to remove the `tracing-fmt` filter types entirely.